src/eddington/exceptions.py

```diff
@@ -63,3 +63,9 @@ def __init__(self, column):  # noqa: D107
 
 class FitDataColumnsSelectionError(FitDataError):  # noqa: D101
     pass
+
+
+class FitDataInvalidSyntax(FitDataError):  # noqa: D101
+    def __init__(self, col, row, value):  # noqa: D107
+        msg = f'the cell at column:"{col}", row:"{row}" has invalid syntax{value}.'
```

Comment:
It is kind of a convension that the row is printed first and then the column. Please switch them.